tests/standalone_tests/test_guard.py

```diff
@@ -105,6 +107,27 @@ def test_is_valid_filter():
     assert not is_valid_filter(
         {"rules": [{"id": "name1"}, {"rules": [{"id": "name.2"}]}]}
     )
+    assert not is_valid_filter(
+        {
+            "condition": "AND",
+            "rules": [
+                {
+                    "condition": "OR",
+                    "rules": [
+                        {
+                            "id": 'col1"=3)); drop table lala; --"l3',
```

Comment:
Maybe limit that only to an included double quote? The rest are valid name right?